composer.json

```
@@ -22,7 +22,7 @@
         "psr/http-message": "^1.0",
         "psr/http-client": "^1.0",
         "php-http/httplug": "^1.1 || ^2.0",
-        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0",
+        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0" || ^6.0",
```

There's a breaking typo here. Here's the fix.
```suggestion
        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```